### DISCUSSION TITLE
Documentation has typo in artifact name "bucket4j-dynamodb-sdk-v1"

### DISCUSSION THREAD

**Discussion by WisdomMillsOwoo** [vladimir-bukhtoyarov/bucket4j#379] [General]
The README located [here](https://github.com/bucket4j/bucket4j/blob/master/README.md) has the entry `bucket4j-bucket4j-dynamodb-sdk-v1`, I believe it should be just `bucket4j-dynamodb-sdk-v1`

**Comment by vladimir-bukhtoyarov** [comment]
fixed
